src/shared/dropdown-wrapper/element-with-dropdown.tsx

```diff
@@ -76,13 +77,11 @@ export const ElementWithDropdown = ({
         duration: 300,
     });
 
-    const click = useClick(context, { enabled });
-    const focus = useFocus(context, { enabled });
```

Comment:
removed to give control to the parent

there may be cases where the main element is focused but popover should remain closed (e.g. toggling behaviour)